reactor-core/src/main/java/reactor/core/publisher/Sinks.java

```diff
@@ -605,7 +631,11 @@ public Emission getReason() {
 		 *
 		 */
 		@Deprecated
-		void emitNext(T t);
+		default void emitNext(T t) {
+			emitNext(t, EmitFailureHandler.FAIL_FAST);
+		}
+
+		void emitNext(T t, EmitFailureHandler failureHandler);
```

Comment:
missing javadoc. insist on the fact that this should rely on `tryEmitNext` (see https://github.com/reactor/reactor-core/pull/2405#discussion_r496589276)

also, big red warning that the method can throw an `EmissionException` (see https://github.com/reactor/reactor-core/pull/2405#discussion_r496559468)